plugins/conventional-commits/src/index.ts

```diff
@@ -135,6 +135,56 @@ export default class ConventionalCommitsPlugin implements IPlugin {
       return this.storedGetBump(message);
     };
 
+    auto.hooks.prCheck.tapPromise(this.name, async ({ pr }) => {
+      if (!auto.git) {
+        return;
+      }
+
+      const VERSIONS = [
+        SEMVER.major,
+        SEMVER.minor,
+        SEMVER.patch,
+        "skip",
+      ] as const;
```

Comment:
This won't work since a user can configure different labels to act as minor/major/patch. You need to use `auto.semVerLabels` which is a map from release type to the labels that equate to that.

Reply:
this wont work specifically for the label already on the PR checking correct? That array is exactly what was already defined in the `getBump` function that is used for the rest of the plugin?